src/components/DayPicker.jsx

```diff
@@ -371,8 +375,10 @@ export default class DayPicker extends React.Component {
       this.props.onPrevMonthClick(e);
     }
 
-    const translationValue =
-      this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
+    let translationValue = this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
+    if (this.props.isRTL && this.isHorizontal()) {
+      translationValue = (-this.dayPickerWidth + this.state.translationValueRTL);
```

Comment:
`translationValue = this.state.translationValueRTL - this.dayPickerWidth;` should work here? (also it'd be ideal to destructure all props and state at the top of the function)